queries.sql

```diff
@@ -14,4 +14,79 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg  >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg  >= 10.4 AND weight_kg <= 17.3;
+
+/* Part 1 */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
```

Comment:
- [ ] you should verify if the changes were made before and after rollback as well

Reply:
<img width="643" alt="1" src="https://user-images.githubusercontent.com/53530780/189983930-ce344505-16c5-4d4a-9563-119fe0726083.png">